spqlios/reim4/reim4_fftvec_addmul_ref.c

```diff
@@ -0,0 +1,97 @@
+#include <errno.h>
+#include <malloc.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "../commons_private.h"
+#include "reim4_fftvec.h"
+#include "reim4_fftvec_private.h"
+
+void* init_reim4_fftvec_addmul_precomp(REIM4_FFTVEC_ADDMUL_PRECOMP* res, uint32_t m) {
+  res->m = m;
+  if (CPU_SUPPORTS("fma")) {
+    if (m >= 2) {
```

Comment:
the asm loop seems to take elements 8 by 8, I think we need m>=8

Reply:
oops, 4 re and 4im, so m>=4

Reply:
fixed!